js/cid/cid.js

```diff
@@ -1,33 +1,9 @@
+var cid = require('can-cid');
 /**
- * @module {function} can-util/js/cid/cid cid
+ * @module can-util/js/cid/cid
  * @parent can-util/js
- * @signature `cid(object, optionalObjectType)`
- *
- * Get a unique identifier for the object, optionally prefixed by a type name.
- *
- * Once set, the unique identifier does not change, even if the type name
- * changes on subsequent calls.
- *
- * ```js
- * var cid = require("can-util/js/cid/cid");
- * var x = {};
- * var y = {};
- * 
- * console.log(cid(x, "demo")); // -> "demo1"
- * console.log(cid(x, "prod")); // -> "demo1"
- * console.log(cid(y));         // -> "2"
- * ```
- *
- * @param {Object} object The object to uniquely identify.
- * @param {String} name   An optional type name with which to prefix the identifier 
- *
- * @return {String} Returns the unique identifier
+ * @description Deprecated. Use [can-cid] instead.
  */
-var cid = 0;
-module.exports = function (object, name) {
-	if (!object._cid) {
-		cid++;
-		object._cid = (name || '') + cid;
-	}
-	return object._cid;
-};
+console.warn('can-util/js/cid/cid is deprecated, use can-cid instead');
```

Comment:
Couple of things here:

1. IE9 still has the issue where it the `console` doesn't exist unless devtools are open. I ran into this when setting up stealjs/steal tests.
2. I don't think we should give warnings for APIs that users are (very much likely) not using. This should be communicated some other way to library maintainers (99.99% us) than in user's browsers.